packages/astro/src/content/loaders/types.ts

```diff
@@ -3,7 +3,9 @@ import type { ZodSchema } from 'zod';
 import type { AstroIntegrationLogger } from '../../core/logger/core.js';
 import type { AstroConfig } from '../../types/public/config.js';
 import type { ContentEntryType } from '../../types/public/content.js';
-import type { MetaStore, ScopedDataStore } from '../mutable-data-store.js';
+import type { AstroDataStore, MetaStore } from '../mutable-data-store.js';
+
+export type { AstroDataStore, MetaStore };
```

Comment:
Would it be weird that there's `AstroDataStore` and `MetaStore`, and someone would wonder why not `DataStore` or `AstroMetaStore`?

Personally I'd lean on only `DataStore`, but that conflicts with the actual base `DataStore` class and maybe needs more refactors. Or do you think this shouldn't be a problem?

Reply:
Yeah, you're right. I've renamed it and done the refactor